.phpstorm.meta.php

```diff
@@ -0,0 +1,7 @@
+<?php
+// .phpstorm.meta.php
+
+namespace PHPSTORM_META {
+
+    override(\Laravel\Dusk\Browser::visit(0), type(0));
```

Comment:
This is a weird trick to improve a bit the developer experience with [page methods](https://laravel.com/docs/10.x/dusk#page-methods).

Basically with this PhpStorm doesn't complain about undefined method, when doing `visit(new SomePage)` followed by some methods define in SomePage, here is an example from LoginPageTest: 
![image](https://github.com/brendt/rfc-vote/assets/5923843/89d659af-3ca0-4fb0-b592-b0fc310ed139)

@brendt please let me know how you feel about this, it is a bit unconventional and I wasn't able to find a better way a more general way of improving the page methods.

Reply:
I'm totally fine with it. I use the Laravel Idea plugin, which I think supports this out of the box, but I'm fine adding it via phpstorm meta as well 👍

Reply:
I use Laravel Idea plugin as well, but I wasn't able to make it work :)